### PR TITLE
Feat: add special `angular` detector

### PR DESCRIPTION
**Pull request by rjlopezdev**
# Motivation
Angular projects can contains hidden dependencies over `angular.json` & `angular-cli.json` file.
This dependencies could be `scripts`, `styles` or `assets` from `node_modules` folder.

# Resolution

I code a simply detector for checking this file & return used dependencies.

# ⚠️  Help wanted 

I coded a skeleton for use cases I consider fits. Do you think that it fills properly? If agree, I'll implement it 😸

### CODE REVIEW

**Comment by znarf** [comment]
@rjlopezdev Looks good. Could you give some sample repositories to try that?

Tests descriptions are looking good too. I'm not the main maintainer but I see no blocker for a merge once they're there.

I'm surprised coverage doesn't seems affected yet according to codecov. The goal is normally to keep the same level of coverage as today.

File: src/special/angular.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      }
+      traverse(obj[k], candidates);
+    }
+  });
+};
+
+export default async function parseAngular(filename) {
+  const basename = path.basename(filename);
+
+  if (ANGULAR_FILES.includes(basename)) {
```
Comment (LinusU) [inline src/special/angular.js:21]: This function could potentially be a bit cleaner with an early return:

```js
  if (!ANGULAR_FILES.includes(basename)) return false;
```

**Comment by rumpl** [comment]
@rjlopezdev thaks for the PR, code looks good, we would just need some tests :)

**Comment by rjlopezdev** [comment]
Sorry for delay, I'm so busy. I will add some other cases for angular projects & test It ASAP ❤️

**Comment by rumpl** [comment]
No rush ;) just wanted to know if you still plan on working on this. Thanks for the PR :)

**Comment by rumpl** [comment]
Ping @rjlopezdev :)

Sorry to bother you, do you still want to do this? I can carry it if you don't have the time

Cheers

**Pull request closed without merging**